lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -218,6 +238,7 @@ public function scopeTimeStamp($scope = null)
         @date_default_timezone_set($timezone);
         $date = date('Y-m-d H:i:s');
         @date_default_timezone_set($currentTimezone);
+
```

Comment:
Looks like some new lines are automatically added by your IDE, please avoid this.